test/bootstrap_fields_test.rb

```diff
@@ -201,6 +202,17 @@ class BootstrapFieldsTest < ActionView::TestCase
       </form>
     HTML
     assert_equivalent_xml expected, output
+
+    ## TODO: DRY up the tests for the _with version
+    if Gem::Version.new(::Rails::VERSION::STRING).release >= Gem::Version.new('5.1.0')
```

Comment:
Unless we have reason to believe that `fields` works fundamentally differently than `fields_for`, then I am not sure there is much value for the new tests in this file.